GitCommands/Settings/AppSettings.cs

```diff
@@ -1645,16 +1579,13 @@ public static DiffListSortType DiffListSorting
 
         public static string GetGitExtensionsFullPath()
         {
-            return Application.ExecutablePath;
+            return _applicationExecutablePath;
         }
 
         [CanBeNull]
         public static string GetGitExtensionsDirectory()
         {
-            var assembly = Assembly.GetEntryAssembly() ?? // common case, GitExtensions.exe
-                           Assembly.GetExecutingAssembly(); // unit test, GitCommands.dll
-            var path = assembly.Location;
-            return Path.GetDirectoryName(path);
+            return Path.GetDirectoryName(GetGitExtensionsFullPath());
```

Comment:
Tests not affected?

Reply:
Tests were deleted with the code that was tested: `ThemeManager`, `ThemeManagerTests`.